binding/go/rhino.go

```diff
@@ -125,10 +125,18 @@ type Rhino struct {
 	// Sensitivity should be a floating-point number within 0 and 1.
 	Sensitivity float32
 
+	/// Endpoint duration in seconds. An endpoint is a chunk of silence at the end of an
```

Comment:
so for multi lines comments, we need to use three slashes in Go?

Reply:
Nope, copy/paste error